src/components/Contacts/ContactListTableMobile.jsx

```diff
@@ -0,0 +1,199 @@
+import MoreVertIcon from '@mui/icons-material/MoreVert';
+import SendIcon from '@mui/icons-material/Send';
+import Typography from '@mui/material/Typography';
+import { useTheme } from '@mui/material/styles';
+// Custom Hooks Imports
+import useNotification from '@hooks/useNotification';
+// Util Imports
+import { saveToClipboard } from '@utils';
+// Component Imports
+import ContactProfileIcon from './ContactProfileIcon';
+
+/**
+ * @typedef {object} Contact
+ * @property {string} webId - The Web ID of the contact
+ * @property {string} givenName - The given name of the contact
+ * @property {string} familyName - The family name of the contact
+ */
+
+/**
+ * ContactListTableMobile - Component for displaying contacts in a DataGrid
+ *
+ * @memberof Contacts
+ * @name ContactListTableMobile
+ * @param {object} Props - The props for ContactListTableMobile
+ * @param {Contact[]} Props.contacts - The list of contacts to display
+ * @param {Function} Props.deleteContact - Function to delete a contact
+ * @param {Function} Props.handleSendMessage - Function to handle sending a message
+ * @returns {React.JSX.Element} The ContactListTableMobile component
+ */
+const ContactListTableMobile = ({ contacts, deleteContact, handleSendMessage }) => {
+  const { addNotification } = useNotification();
+  const [anchorEl, setAnchorEl] = useState(null);
+  const [openMenu, setOpenMenu] = useState(null);
+  const theme = useTheme();
+
+  const handleClick = (event, contact) => {
+    setAnchorEl(event.currentTarget);
+    setOpenMenu(contact.webId);
+  };
+  const handleClose = () => {
+    setAnchorEl(null);
+    setOpenMenu(null);
+  };
+
+  const handleMenuItemClick = (action, contact) => () => {
+    action(contact);
+    handleClose();
+  };
+
+  const handleProfileClick = (contactData) => {
+    <ContactProfileIcon contact={contactData} />;
+  };
+
+  const iconSize = {
+    height: '24px',
+    width: '24px'
+  };
+
+  const iconStyling = {
+    width: '100%'
+  };
+
+  return (
+    <Box>
+      <Box
+        sx={{
+          my: '15px',
+          p: '15px',
+          background: theme.palette.primary.light,
+          color: '#fff',
+          borderRadius: '8px',
+          position: 'relative'
+        }}
+      >
+        <Box>
+          <Typography>Name</Typography>
+          <Box
+            sx={{
+              position: 'absolute',
+              top: '50%',
+              right: 15,
+              transform: 'translateY(-50%)'
+            }}
+          >
+            <Typography>Actions</Typography>
+          </Box>
+        </Box>
+      </Box>
+      {contacts?.map((contact) => (
+        <Box key={contact.webId}>
+          <Card
+            sx={{
+              my: '5px',
+              position: 'relative'
+            }}
+          >
+            <CardContent>
+              <Box>
+                <Typography variant="body1" component="div" noWrap>
+                  {contact.givenName || ''} {contact.familyName || ''}
```

Comment:
I think adding a non-breaking space between the given and family name would let the line from collapsing if the user ends up not adding names to their profiles.

```
{contact.givenName || ''}&nbsp;{contact.familyName || ''}
```

![Screenshot 2024-06-25 at 9 49 47 PM](https://github.com/codeforpdx/PASS/assets/14917816/24273dde-c298-4af0-b01b-6eb66422e91b)

Reply:
For me it's not very visually different (now it seems too large rather than too small) but I've added it anyway. 

Maybe we should have a default for when they don't add a name, like "No name provided" that is slightly greyed out. Not sure.

Reply:
The size should be the same as if there were texts for the name, so it's fine

Adding no name provided is okay too

Reply:
For reference, here's what it looks like with `{contact.givenName || ''} {contact.familyName || ''}`:

<img width="240" alt="2024-06-26 (5)" src="https://github.com/codeforpdx/PASS/assets/83156697/23eb0e7c-ec0a-4cbc-a0a3-7e399f85d2ee">

And with `{contact.givenName || ''}&nbsp;{contact.familyName || ''}`:

<img width="238" alt="2024-06-26 (4)" src="https://github.com/codeforpdx/PASS/assets/83156697/5504b223-4cb9-4679-999d-64dc3cbef084">

Will now look into adding some kind of placeholder.

Reply:
A bit confused for the reference pic?

The gap with the non breaking space looks the same as the one I've screenshootes in an earlier comment

Reply:
Oh, I'm talking about the size of the card itself, not the space in between names.